### PR TITLE
Added ssl certs to kube-config google.

### PR DESCRIPTION
**Pull request by piosz**
As requested in https://github.com/GoogleCloudPlatform/kubernetes/pull/12098

### CODE REVIEW

**Comment by vishh** [comment]
ok to test

File: deploy/kube-config/google/heapster-controller.json [hunk 1]
```diff
@@ -24,6 +24,14 @@
 		}
 	    },
 	    "spec": {
+		"volumes": [
+		    {
+			"name": "ssl-certs",
+			"hostPath": {
+			    "path": "/etc/ssl/certs"
```
Comment (jimmidyson) [inline deploy/kube-config/google/heapster-controller.json]: Does this work on non-debian based distros?

**Comment by ScOut3R** [comment]
I have tried this on CoreOS and it does not work. I changed the path to "/usr/share/ca-certificates" and that works. I assume the symlinks under /etc/ssl/certs aren't resolvable inside the container.

**Comment by vishh** [comment]
cc @mikedanese

**Pull request merged**